<?xml version="1.0"?>
<launch>
	<node pkg="cv_camera" type="cv_camera_node" name="cv_camera" output="screen">
		<param name="image_width" value="640" />
		<param name="image_height" value="360" />
	</node>

    <param name="package_name" value="operation_test" />

    <!-- robotis op3 manager -->
    <include file="$(find op3_manager)/launch/op3_manager.launch"/>



	<node pkg="operation_test" type="image_test.py" name="image_test" output="screen">
		<param name="test_param" value="It's Tsen callinggggggggggggggg." />			
	</node>

</launch>

